packages/core/crates/core-crypto/Cargo.toml

```diff
@@ -0,0 +1,54 @@
+[package]
+name = "core-crypto"
+version = "0.1.0"
+description = "Core cryptographic primitives and functions used in HOPR"
+authors = ["HOPR Association <tech@hoprnet.org>"]
+edition = "2021"
+homepage = "https://hoprnet.org/"
+repository = "https://github.com/hoprnet/hoprnet"
+license = "LGPL-3.0-only"
```

Comment:
Why this license?

Reply:
Possibly some default, which should I use?

Reply:
Update all crates to `GPL-3.0-only`